src/libs/getPlaidLinkTokenParameters/types.ts

```diff
@@ -0,0 +1,14 @@
+type PlaidLinkTokenParameters = {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    android_package?: string;
+
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    redirect_uri?: string;
+
+    allowDebit?: boolean;
+    bankAccountID?: number;
```

Comment:
Should we add captions for these properties?

Reply:
Since we don't have a requirement to caption TS types, and this type is not widely used, I think we can leave it as is.